### PR TITLE
Prevent button text wrapping or truncation

### PR DESCRIPTION
**Pull request by mdmathias**

### CODE REVIEW

File: Samples/Swift/DaysUntilBirthday/Shared/Views/SignInView.swift [hunk 1]
```diff
@@ -24,8 +24,8 @@ struct SignInView: View {
     VStack {
       HStack {
         let buttonViewModel = GoogleSignInButtonViewModel(
-          scheme: .light,
-          style: .standard
+          scheme: .dark,
```
Comment (petea) [inline Samples/Swift/DaysUntilBirthday/Shared/Views/SignInView.swift:27]: Did you intend to commit this?

File: GoogleSignInSwift/Sources/GoogleSignInButtonStyling.swift [hunk 2]
```diff
@@ -222,6 +222,30 @@ extension GoogleSignInButtonStyle {
     case .icon: return ""
     }
   }
+
+  var minWidthForButtonText: CGFloat {
+    let bt = buttonText as NSString
+#if os(iOS) || targetEnvironment(macCatalyst)
+    let font = UIFont(name: fontNameRobotoBold, size: fontSize)
+    let rect = bt.boundingRect(
```
Comment (petea) [inline GoogleSignInSwift/Sources/GoogleSignInButtonStyling.swift:230]: It looks like the signature for boundingRect is pretty similar on macOS.  Could we dry this up?

File: Samples/Swift/DaysUntilBirthday/Shared/Views/SignInView.swift (hunk 1 above)
Comment (mdmathias) [reply Samples/Swift/DaysUntilBirthday/Shared/Views/SignInView.swift:27]: Gah, no!

File: GoogleSignInSwift/Sources/GoogleSignInButtonStyling.swift (hunk 2 above)
Comment (mdmathias) [reply GoogleSignInSwift/Sources/GoogleSignInButtonStyling.swift]: Yep!

**Review by petea — APPROVED** [review]

**Pull request merged**